sklearn/calibration.py

```diff
@@ -141,6 +140,11 @@ class CalibratedClassifierCV(ClassifierMixin,
     classes_ : ndarray of shape (n_classes,)
         The class labels.
 
+    n_features_in_ : int
+        Number of features seen during :term:`fit`.
+
+        .. versionadded:: 0.24
```

Comment:
```suggestion
        .. versionadded:: 1.0
```

Reply:
The attribute was already there in 0.24:

```python
>>> import sklearn; sklearn.__version__
'0.24.1'
>>> from sklearn.datasets import make_classification
>>> X, y = make_classification()
>>> from sklearn.calibration import CalibratedClassifierCV
>>> cccv = CalibratedClassifierCV().fit(X, y)
>>> cccv.n_features_in_
20
```